site/en/blog/immutable-document-domain/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "`document.domain` is becoming immutable"
+description: >
+  Websites will be unable to set `document.domain` on Chrome starting in version 101. If your website relies on setting `document.domain`, your action is required.
+subhead: >
+  Beginning with Chrome version 101, websites will be unable to set `document.domain`.
+date: 2022-01-06
+authors:
+  - agektmr
+tags:
+  - security
+hero: 'image/YLflGBAPWecgtKJLqCJHSzHqe2J2/grGMzuhOjsGhN150dONe.jpg'
+alt: >
+  A dog in disguise.
+---
+
+**If your website relies on relaxing the same-origin policy via `document.domain`,
+your action is required.**
+
```

Comment:
Add a quick definition of what `document.domain` does and link to MDN docs at the first mention.

Reply:
```suggestion

[`document.domain`](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain)
was designed to get or set the origin's hostname.

```

Reply:
Two things:
- I think it's useful to define what it was designed to do, rather than what it actively does because what it does is changing.
- I initially thought they `key-term` aside would be appropriate, but I think that would draw attention away from the warning, so just having the definition means it may make sense to move this sentence to after the first sentence of the next paragraph
   > Beginning with Chrome version 101, websites will be unable to set [`document.domain`](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain). `document.domain` was designed to get or set the origin's hostname. Websites will need to use alternative approaches such as `postMessage()` or Channel Messaging API to communicate cross-origin.
   > 
   > If a website relies on same-origin policy relaxation via `document.domain` to function correctly, the site will need to send an `Origin-Agent-Cluster: ?0` header along with all documents that require that behavior (note that `document.domain` has no effect if only one document sets it).

Reply:
Yep, that works great, you got this under control! :)